app/ui/src/app/integration/integration_detail/integration-detail.component.html

```diff
@@ -1,7 +1,13 @@
 <syndesis-modal [title]="modalTitle" [message]="modalMessage" [type]="modalType" [primaryText]="modalPrimaryText">
 </syndesis-modal>
-<syndesis-loading [loading]="loading">
-  <div *ngIf="integration" class="integration-detail syn-scrollable">
+
+<div *ngIf="loading; else content">
```

Comment:
The `syndesis-loading` component should be kinda doing this already, was it not working for this case?

Reply:
Gotcha!  Didn't realize that, can you raise a bug against the loading component?  We should try and improve it so that it's template isn't evaluated until the loading input is false, that's the intent anyways.

Reply:
Sure! I tried to use the same logic inside the loading component itself, but as I expected it doesn't solve the issue.

If we were using React, I'd know how to fix this, but with Angular, I'm a bit at a loss.